examples/warehouse/domain/README

```diff
@@ -41,3 +45,10 @@ dns.yaml:
       ns:
         - IP address of NS server authoritative for this domain
         - ...
+
+services.yaml:
+  domain:
+    services:
+      syslog:
+        - A central Syslog server, in the form <fqdn>:<port>
+        - ...
```

Comment:
Actually, that's precisely the format that Rsyslog wants, so I wrote it like that to minimise string parsing. But you're right, it should be more general.